### PR TITLE
[VIX-2020] Announcing devopsdays Vitoria

### PR DESCRIPTION
**Pull request by sprt-balarini**
*PR regarding devopsdays event in the city of Brazil ES-Vitoria in 2020`*

### CODE REVIEW

File: data/events/2020-vitoria.yml [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+description: "Devopsdays is coming to Vitoria!" # Edit this to suit your preferences
+ga_tracking_id: "" # If you have your own Google Analytics tracking ID, enter it here. Example: "UA-74738648-1"
+
+# All dates are in unquoted 2020-MM-DDTHH:MM:SS+TZ:TZ, like this:
+#   variable: 2019-01-04T00:00:00+02:00
+#   variable: 2019-01-05T23:59:59+02:00
+# Note: we allow 2020-MM-DD for backward compatibility, but it can lead to unexpected behaviors (like your event disappearing from the front page during your last day)
+
+startdate:  2020-02-08T00:00:00+13:00 # The start date of your event. Leave blank if you don't have a venue reserved yet.
+enddate:  2020-02-08T00:00:00+18:00 # The end date of your event. Leave blank if you don't have a venue reserved yet.
```
Comment (bridgetkromhout) [inline data/events/2020-vitoria.yml]: Please fix the TZ offset and set the end time later than the start time.

File: data/events/2020-vitoria.yml [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+team_members: # Name is the only required field for team members.
+  - name: "Giovani Balarini"
+    twitter: "GBalarini"
+    employer: "IT analyst"
+    bio: ""
+    github: ""
+    facebook: ""
+    linkedin: "https://br.linkedin.com/in/giovani-balarini-a4a25a86"
+    website: ""
+    image: "giovani balarini.jpg"
```
Comment (bridgetkromhout) [inline data/events/2020-vitoria.yml]: Please fix all the images to have `-` instead of ` ` (space).

File: deploy.sh [hunk 3]
```diff
@@ -1,14 +0,0 @@
-hugo -v # command to build your site
```
Comment (bridgetkromhout) [inline deploy.sh]: Please just remove this file from the PR - thanks.

**Comment by bridgetkromhout** [comment]
Please don't put any files in `utilities/img`.

File: deploy.sh (hunk 3 above)
Comment (MikeRosTX) [reply deploy.sh]: Marking this as resolved, because it looks like it was deleted.

**Comment by bridgetkromhout** [comment]
Looking better - please restore deploy.sh though - thanks!

**Comment by MikeRosTX** [comment]
The **_Sponsor_** levels were removed. 
@sprt-balarini is Vitoria planning on having Sponsors? 
I think the answer is **_Yes_**, because you have a Sponsors page.

**Comment by sprt-balarini** [comment]
> The **_Sponsor_** levels were removed.
> @sprt-balarini is Vitoria planning on having Sponsors?
> I think the answer is **_Yes_**, because you have a Sponsors page.

In this moment no.
We do not decide values for the sponsors.
Immediately we are rushing to submit the
event in our city.

**Comment by MikeRosTX** [comment]
Defina para **_não fundir_** até que um e-mail seja enviado para a equipe principal com uma lista de todos os nomes completos e e-mails do organizador de Vitoria.

**Comment by sprt-balarini** [comment]
> Defina para **_não fundir_** até que um e-mail seja enviado para a equipe principal com uma lista de todos os nomes completos e e-mails do organizador de Vitoria.

Muito obrigado por adicionar o deploy.sh

O nosso proximo passo então seria encaminhar um email com todos os organizadores com os emails.
Para qual email podemos encaminhar?

**Comment by sprt-balarini** [comment]
> Defina para **_não fundir_** até que um e-mail seja enviado para a equipe principal com uma lista de todos os nomes completos e e-mails do organizador de Vitoria.

Thanks so much for adding deploy.sh
Our next step would then be to forward an email with all the organizers with the emails.
Which email can we forward to?

**Comment by MikeRosTX** [comment]
Sim, está correcto. Envie-o para **_info@devopsdays.org_**.

**Comment by sprt-balarini** [comment]
> Sim, está correcto. Envie-o para **_[info@devopsdays.org](mailto:info@devopsdays.org)_**.

Obrigado Mike.
Estamos encaminhando novamente o email.

**Comment by gomex** [comment]
@sprt-balarini can you please restore the deploy.sh? In this case, just add a blank line in the end of file, ok?

**Pull request merged**